types/audit.d.ts

```diff
@@ -64,23 +70,31 @@ declare global {
     export interface Product {
```

Comment:
somewhat unrelated, but instead of duplicating should this `Pick<>` + extend from `Result`?

Reply:
> somewhat unrelated, but instead of duplicating should this `Pick<>` + extend from `Result`?

yeah, I was thinking about the same thing while doing this, I think it's worth trying to make it work.